20230706(Johnny)

JohnnyHuang1104 opened the pull request:
UI Modified - 
(1)Add pageIndicator to activity_main.xml
(2)Add left and right arrow button to page.xml, but can't click.
(3)Add Toolbar to activity_main.xml

JohnnyHuang1104 committed:
UI Modified -
(1)Add pageIndicator to activity_main.xml
(2)Add left and right arrow button to page.xml, but can't click.
(3)Add Toolbar to activity_main.xml

Wujuichen commented on app/src/main/java/com/example/javaviewpager2/MainActivity.java:
```diff
@@ -6,29 +6,45 @@
+        setViewPagerTransformerEnlargeWhenScroll(0.2f,40);
+
+        // 將PageIndicator與ViewPager2結合(attach)
+        linkPageIndicatorAndViewPager2();
```
這是一個本身就存在的函數?

Wujuichen replied:
這是一個本身就存在的函數?

Wujuichen commented on app/src/main/java/com/example/javaviewpager2/MainActivity.java:
```diff
@@ -6,29 +6,45 @@
+        linkPageIndicatorAndViewPager2();
+
 
     }
```
可以上移一格

JohnnyHuang1104 replied:
好的

Wujuichen commented on app/src/main/java/com/example/javaviewpager2/MainActivity.java:
```diff
@@ -6,29 +6,45 @@
+        setViewPagerTransformerEnlargeWhenScroll(0.2f,40);
+
+        // 將PageIndicator與ViewPager2結合(attach)
+        linkPageIndicatorAndViewPager2();
```
這是一個本身就存在的函數?

JohnnyHuang1104 replied:
linkPageIndicatorAndViewPager2(); 這行是我自定義的包裝函數 在更下方的程式碼(112~122行)有這部分的敘述

Wujuichen commented on app/src/main/res/layout/activity_main.xml:
```diff
@@ -7,11 +7,26 @@
+        android:layout_weight="9"
         android:paddingHorizontal="35dp"
-        android:paddingVertical="35dp" />
+        android:layout_marginTop="15dp"
```
建議/> 上移一格

Wujuichen commented on app/src/main/res/menu/menu.xml:
```diff
@@ -0,0 +1,15 @@
+    <item
+        android:id="@+id/style_move"
+        android:title="Move"
+        app:showAsAction="never"/>
```
showAsAction 是甚麼意思??

Wujuichen commented on app/src/main/res/drawable/tab_selector.xml:
```diff
@@ -0,0 +1,10 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <!--未被滑到的頁面會預設成default_dot,而當前的頁面則會顯示selected_dot  -->
+    <item android:drawable="@drawable/selected_dot"
```
等號後空一格

JohnnyHuang1104 committed:
UI Modified -
(1)Add pageIndicator to activity_main.xml
(2)Add left and right arrow button to page.xml, but can't click.
(3)Add Toolbar to activity_main.xml
